sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundantConditionalAroundAssignment.CSharp9.cs

```diff
@@ -0,0 +1,33 @@
+﻿int x = 5;
+int y = 6;
+
+y = y switch
+{
+    not 5 => 5 // Compliant - FN
```

Comment:
could you also add a test with another switch arm?
how should it behave? raise only on the arm that is redundant, right?

Reply:
I'm not sure I understand how the other switch arm should look like